packages/core/frontend/sirius-components-core/src/workbench/Workbench.types.ts

```diff
@@ -20,6 +20,7 @@ export interface SelectionEntry {
   id: string;
   label: string;
   kind: string;
+  iconURL?: string;
```

Comment:
This interface is one of the most used interface in all of our applications. Why do you need that here because that would be a MAJOR change? Nothing here should ever change without an ADR indicating why that would be necessary and I would be more in favor of removing things from selection entry (label is an error here) than adding new ones.

Reply:
I use it to get the icon in the right panel of the modal in multi-valued reference after the transfer from the left panel.
I'll check if I can do it otherwise.

Reply:
I changed it to get the icons from the item options